File: blackjack/Card.java
package blackjack;

public class Card {

    private int value;
    private String suit;

    public Card(int value, String suit) {
        this.value = value;
        this.suit = suit;
    }

    public int getValue() {
        return this.value;
    }

    public String getSuit() {
        return this.suit;
    }

    public void printCard() {
        if (value == 1) {
            System.out.print("A" + suit);
        } else if (value == 11) {
            System.out.print("J" + suit);
        } else if (value == 12) {
            System.out.print("Q" + suit);
        } else if (value == 13) {
            System.out.print("K" + suit);
        } else {
            System.out.print(value + suit);
        }
    }

}

File: PythonBlackjack/Card.py
class Card:

    def __init__(self, value, suit):
        self.value = value
        self.suit = suit

    def printCard(self):
        if self.value == 1:
            print("A", self.suit, sep="", end=" ")
        elif self.value == 13:
            print("K", self.suit, sep="", end=" ")
        elif self.value == 12:
            print("Q", self.suit, sep="", end=" ")
        elif self.value == 11:
            print("J", self.suit, sep="", end=" ")
        else:
            print(self.value, self.suit, sep="", end=" ")
